portfolio/src/main/webapp/script.js

```diff
@@ -56,6 +56,18 @@ function loadSkills () {
  */
  async function fetchFromData() {
   const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('secret-message').innerText = quote; 
+  // This gives a list of messages
+  const msgs = await response.json();
+
+  // Clear the child elements from the div to avoid duplicating
+  const root = document.getElementById("all-messages");
+  root.innterHTML = ""; 
+
+  // Adds all of the messages to a div
+  msgs.forEach(m => {
+    const node = document.createElement("p"); 
+    node.className = "msg"; 
+    node.innerHTML = m;
```

Comment:
innerHTML is discouraged for security reasons :) 

Seems like you can just use innerText here?

Reply:
ohhh yes that does make sense

Reply:
can I use jquery instead?

Reply:
or textContent?

Reply:
After reading https://api.jquery.com/append/ and it seems like it handles HTML content correctly. I assume it has its own "parsing" to decide whether we are appending a text or dom node. Sorry for the back-and-forth about this, sounds like your original `.append` solution is fine :) 

So for your questions, I think yes, using jquery is fine.